HintList.py

```diff
@@ -363,7 +363,16 @@ def tokens_required_by_settings(world):
     'Rupees (50)':                                              (["big bucks", "a purple gem", "wealth"], "a Purple Rupee", 'item'),
     'Rupees (200)':                                             (["a juicy jackpot", "a yellow gem", "a giant gem", "great wealth"], "a Huge Rupee", 'item'),
     'Weird Egg':                                                (["a chicken dilemma"], "the Weird Egg", 'item'),
+    'Chicken':                                                  (["the Chicken"],"the Chicken",'item'),
     'Zeldas Letter':                                            (["an autograph", "royal stationery", "royal snail mail"], "Zelda's Letter", 'item'),
+    'Keaton Mask':                                              (["the Keaton Mask"],"the Keaton Mask",'item'),
+    'Skull Mask':                                               (["the Skull Mask"],"the Skull Mask",'item'),
+    'Spooky Mask':                                              (["the Spooky Mask"],"the Spooky Mask",'item'),
+    'Bunny Hood':                                               (["the Bunny Hood"],"the Bunny Hood",'item'),
+    'Goron Mask':                                               (["the Goron Mask"],"the Goron Mask",'item'),
+    'Zora Mask':                                                (["the Zora Mask"],"the Zora Mask",'item'),
+    'Gerudo Mask':                                              (["the Gerudo Mask"],"the Gerudo Mask",'item'),
+    'Mask of Truth':                                            (["the Mask of Truth"],"the Mask of Truth",'item'),
```

Comment:
The new items need cryptic hint text.

Reply:
Gerudo Mask -> Feminine Facade?

Reply:
Zora Mask -> Fishy Facade?

Reply:
Mask of Truth -> Factual or Faithful Facade?

Reply:
Goron Mask -> Familial Facade? Because they call Link brother. That one is quite a stretch tho lol.

Reply:
The Chicken could just be “a chicken dilemma”, same as the Weird Egg. They're logically equivalent anyway.

Reply:
Spooky Mask could be Frightening Facade.

Reply:
Keaton Mask → Famous Facade

Reply:
Skull Mask → Forest Facade? Bit of a stretch.

Reply:
Maybe fraternal for Goron Mask?

Reply:
Fraternal is much better!

Reply:
Fatal Facade for Skull Mask? Because skulls are associated with death? It works better for MM's Blast Mask tho lol.

Reply:
Fleshless? Theatrical?